pkg/testutils/backup.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2023 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package testutils
+
+import (
+	"github.com/cockroachdb/cockroach/pkg/util/envutil"
+)
+
+const (
+	defaultBackupBucket              = "cockroachdb-backup-testing"
+	longTTLBackupTestingBucket       = "cockroachdb-backup-testing-long-ttl"
+	backupTestingBucketEnvVar        = "BACKUP_TESTING_BUCKET"
+	backupTestingBucketLongTTLEnvVar = "BACKUP_TESTING_BUCKET_LONG_TTL"
+)
+
+// BackupTestingBucket returns the name of the external storage bucket that
+// should be used in a test run. Most times, this will be the regular public
+// bucket. In private test runs, the name of the bucket is passed through an
+// environment variable.
+func BackupTestingBucket() string {
+	return envutil.EnvOrDefaultString(backupTestingBucketEnvVar, defaultBackupBucket)
```

Comment:
Probably better to have this code be exactly the same as master. Also `EnvOrDefaultString` is for internal (`COCKROACH_*`) vars only, I think this will panic.

Reply:
hrm. the linter was barking at me for not using envutil.... lemme figure out how to quiet the linter